webapp/src/dogma/features/repo/RepoList.tsx

```diff
@@ -1,18 +1,23 @@
+import { ViewIcon, DeleteIcon } from '@chakra-ui/icons';
+import { Wrap, WrapItem, Button } from '@chakra-ui/react';
 import { ColumnDef, createColumnHelper } from '@tanstack/react-table';
 import { formatDistance } from 'date-fns';
 import { DynamicDataTable } from 'dogma/common/components/table/DynamicDataTable';
 import { RepoDto } from 'dogma/features/repo/RepoDto';
+import Link from 'next/link';
 
 export type RepoListProps<Data extends object> = {
   data: Data[];
-  name: string;
+  projectName: string;
 };
 
-const RepoList = <Data extends object>({ data, name }: RepoListProps<Data>) => {
+const RepoList = <Data extends object>({ data, projectName }: RepoListProps<Data>) => {
   const columnHelper = createColumnHelper<RepoDto>();
   const columns = [
     columnHelper.accessor((row: RepoDto) => row.name, {
-      cell: (info) => info.getValue(),
+      cell: (info) => (
+        <Link href={`/app/projects/${projectName}/repos/${info.getValue()}`}>{info.getValue()}</Link>
```

Comment:
`next/link` does not seem to render an underline when a cursor is on the link. That would not offer a good UX. I tried to test the chakra-UI `Link` component and it rendered an underline to the link.
How about applying this component in the next PR? https://chakra-ui.com/docs/components/link#usage-with-nextjs